src/App.js

```diff
@@ -0,0 +1,175 @@
+import './App.css';
+import React from 'react';
+import GameField from './GameField';
+import Settings from './Settings';
+import Play from './Play';
+import Intro from './Intro';
+import rs_school_js from './rs_school_js.svg';
+import Timer from './Timer';
+import swapNumbers from './swapNumbers.js';
+import defineMoves from './defineMoves.js';
+import clickSound from './clickSound';
+import { VscGithubAlt } from "react-icons/vsc";
+import playWin from "./playWin.js";
+
+// import getLocalStorage from './getLocalStorage';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);    
+    this.updateSettings = this.updateSettings.bind(this);
+    this.handleStart = this.handleStart.bind(this);
+    this.handleSwap = this.handleSwap.bind(this);
+    this.closeModal = this.closeModal.bind(this);
+    this.handleDemo = this.handleDemo.bind(this);
+    this.state = {
+      modalStyle: {display: 'block'},
+      sound: 'off',
+      mode: 'default',
+      stats: 'counts',
+      theme: 'light',
+      numbers: [1, 2, 3, 4, 5, 6, 7, 8, null],
```

Comment:
I think you need a constant for this array. You use this value in many places. Try to think about other values that could be used as constants